sites/public/pages/finder.tsx

```diff
@@ -0,0 +1,211 @@
+
+const getTranslationString = (str: string) => {
+  if (str === "studio") {
+    return "studioPlus"
+  } else if (str === "oneBdrm") {
+    return "onePlus"
+  } else if (str === "twoBdrm") {
+    return "twoPlus"
+  } else if (str === "threeBdrm") {
+    return "threePlus"
+  } else if (str === "fourBdrm") {
+    return "fourPlus"
+  }
+}
+
+interface finderField {
+  label: string
+  translation: string
+  selected: boolean
+}
+
+interface finderQuestion {
+  fieldGroupName: string
+  fields: finderField[]
+}
+
+interface finderForm {
+  [key: number]: finderQuestion
+}
+
+const Finder = () => {
+  const [currentStep, setCurrentStep] = useState<number>(1)
+  const [completedSteps, setCompletedSteps] = useState<number>(0)
+  const [activeQuestion, setActiveQuestion] = useState<finderQuestion>(null)
+  const formData: finderForm = {}
+
+  const onSubmit = (data: ListingFilterState) => {
+    void router.push(`/listings/filtered?page=${1}&limit=${8}${encodeToFrontendFilterString(data)}`)
+  }
+  useEffect(() => {
+    const getAndSetOptions = async () => {
+      try {
+        const response = await axios.get(`${process.env.backendApiBase}/listings/meta`)
+        // console.log(response.data)
+        if (response.data) {
+          if (response.data.unitTypes) {
+            const bedroomFields = response.data.unitTypes.map((elem) => ({
+              label: elem.name,
+              translation: getTranslationString(elem.name),
+              selected: false,
+            }))
+            formData[1] = { fieldGroupName: "bedRoomSize", fields: bedroomFields }
+          }
+          setActiveQuestion(formData[1])
+        }
+      } catch (e) {
+        console.error(e)
+      }
+    }
+    void getAndSetOptions()
+  }, [])
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { register, handleSubmit, getValues } = useForm()
+
+  const ProgressHeader = (currentStep: number, completedSteps: number) => {
+    return (
+      <div className="flex flex-col w-full">
+        <div className="flex flex-row justify-between">
+          <div className="md:text-xl capitalize font-bold">
+            {t("listingFilters.buttonTitleExtended")}
+          </div>
+          <StepHeader
+            currentStep={currentStep}
+            totalSteps={3}
+            stepPreposition={"of"}
+            stepLabeling={["steps"]}
+          ></StepHeader>
+        </div>
+        <ProgressNav
+          currentPageSection={currentStep}
+          completedSections={completedSteps}
+          labels={["Housing Needs", "Accessibility", "Building Types"]}
+          mounted={true}
+          style="bar"
+        ></ProgressNav>
+      </div>
+    )
+  }
+
+  const nextQuestion = () => {
+    const userSelections = getValues()[formData[currentStep]["fieldGroupName"]]
+    formData[currentStep]["fields"].forEach(
+      (field) => (field["selected"] = userSelections.includes(field))
+    )
+    setCompletedSteps(completedSteps + 1)
```

Comment:
I think for now we don't need the two! Only thing I was considering in keeping both is handling the disclaimer page after all of the steps, but it will definitely be easier to cover that case as part of the later PR.